Create a nice GitHub profile for Almakers (#1)
alma/.github: README.md

<img src="profile/assets/almakers.png" alt="Almakers" width="20%">

## Alma

> At Alma, we put finance at the service of commerce by allowing merchants to increase their sales by 20% on average thanks to payment facilities, without any risk. With Alma, the customer pays later, but the merchant gets paid right away.

Reviewer: ```suggestion
At Alma, we put finance at the service of commerce by allowing merchants to increase their sales by 20% on average. With Alma, the customer pays later, but the merchant gets paid right away with no risk taken.
```

Author: Done. Thanks.
